build_system/dune

```diff
@@ -1,5 +1,11 @@
 (executable
  (name clerk)
  (public_name clerk)
- (libraries catala.runtime catala.utils cmdliner re ANSITerminal)
+ (libraries catala.runtime catala.utils cmdliner re ANSITerminal ninja_utils)
+ (modules clerk)
  (package catala))
+
+(library
+ (name ninja_utils)
```

Comment:
Can you put this one in the `dune-project` at the top-level as a standalone package? Then we'll publish it on opam.